README.md

```diff
@@ -76,6 +76,24 @@ If `Type` of metrics is `uint64` or `uint32` and `Diff` is true, the helper chec
 When differential value is negative, overflow or counter reset may be occurred.
 If the differential value is ten-times above last value, the helper judge this is counter reset, not counter overflow, then the helper set value is unknown. If not, the helper recognizes counter overflow occurred.
 
+## Tempfile
+
+`MackerelPlugin` interface has `Tempfile` field. The Tempfile is used to calc differences in metrics with `Diff: true`.
+If this field is empty, default Tempfile's filename will be generated from plugin filename.
```

Comment:
[nit] s/calc/calculate/, s/empty/omitted/
default Tempfile... => the filename of the temporary file is automatically generated from...